src/smc-hub/kucalc/compute-client.coffee

```diff
@@ -389,6 +389,7 @@ class Project extends EventEmitter
             exclude_history   : undefined
             timeout           : undefined
             bwlimit           : undefined
+            wait_until_done   : 'true'  # by default, wait until done. false only gives the ID to query the status later
```

Comment:
Everything looks good, but I **don't understand** why you made this default a **STRING** `'true'` and not just the boolean `true`?

Reply:
maybe I don't know how to use `curl -X POST -d key=true`, but when I do that, I get a string. I wanted to make this consistent with what's coming in, that's all. The code itself checks for boolean val or the default string.